src/storable.rs

```diff
@@ -492,6 +492,120 @@ where
     };
 }
 
+fn get_size_len<T>() -> usize
```

Comment:
(communication) This method's name doesn't explain to me at all what it's for, and what it's intended purpose is. Please pay attention to these names. It makes much more readable and maintainable.

Possible suggestion:

```suggestion
fn get_num_bytes_required_to_store<T>() -> usize
```